ddtrace/opentracer/tracer.py

```diff
@@ -330,3 +332,25 @@ def extract(self, format, carrier):  # noqa: A002
         dd_span_ctx = ot_span_ctx._dd_context
         self._dd_tracer.context_provider.activate(dd_span_ctx)
         return ot_span_ctx
+
+    def get_correlation_ids(self):
+        # type: () -> Tuple[str, str]
+        """Retrieves the Correlation Identifiers for the current active ``Trace``.
+        This helper method can be achieved manually and should be considered
+        only a shortcut.
+
+        OpenTracing users can still extract these values using the ``ScopeManager``
+        API, though this shortcut is a simple one-liner. The usage is:
```

Comment:
should we document the OpenTracing ScopeManager approach here?

Reply:
Not sure if I'm missing context since this code hasn't been touched in 3 years, but I'm not sure if I'm using the `ScopeManager` API from our OpenTracing instrumentation correctly. 

From OpenTracing docs, the `ScopeManager` should have a callable `active()` method but our OpenTracing instrumentation ScopeManager is a `ThreadLocalScopeManager` that doesn't have a callable but instead a `_ThreadLocalScope` object that leads to the active span. Can you confirm if this is the correct approach?

```
from ddtrace.opentracer.tracer import Tracer
tracer = Tracer(service_name="test")
tracer.start_active_span("ot_span")
active_span = tracer.scope_manager._tls_scope.active._span._dd_span
```